DBZ-3114 Support for Transaction Metadata in MySql connector

nitin456 opened the pull request.

jpechane commented:
@gunnarmorling This is good to go from my PoV.
@nitin456 Please see I've added two commits related to tests and updated a message of the first one.
Please pull the updated branch and push again using email associated with your github account so it is correctl attributed to you. Thanks!

nitin456 commented:
> @gunnarmorling This is good to go from my PoV.
> @nitin456 Please see I've added two commits related to tests and updated a message of the first one.
> Please pull the updated branch and push again using email associated with your github account so it is correctl attributed to you. Thanks!

Thanks for mentoring me in this and fixing the test. I have created new PR, closing this one.
